README.md

```diff
@@ -1,12 +1,14 @@
 # markdownz [![Build Status](https://travis-ci.org/zooniverse/markdownz.svg?branch=master)](https://travis-ci.org/zooniverse/markdownz)
 
-Markdown viewer and editor for the [Zooniverse](https://www.zooniverse.org). Requires React 15.
+Markdown viewer, editor, and help components for the [Zooniverse](https://www.zooniverse.org). Requires React 15.
 
 ## Usage
 
 Available on [npm](http://npmjs.com), include as a dependency using `npm install --save markdownz`
 
-This package contains two publicly accessible components a Markdown viewer and a Markdown editor for Zooniverse-flavored Markdown:
+For the MarkdownHelp component, any default styles can be added to `src/css/markdown-help.css`. Don't forget to add a `require` statement for the css file!
+
+This package contains three publicly accessible components: a Markdown viewer and a Markdown editor for Zooniverse-flavored Markdown, and a MarkdownHelp component that displays a modal illustrating styles and how to implement them.
```

Comment:
This doesn't display a modal. It's just the content. It's up to the dev to decide how to implement.